charts/argo-controller/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v2
+name: argo-controller
+description: Chart for the Argo workflow controller
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
```

Comment:
```suggestion

# This is the version number of the application being deployed. This version number should be
# incremented each time you make changes to the application.
appVersion: 0.0.0

```
Was comparing this to the chart in /pure-ftpd and noticed it had an appVersion.

Reply:
Actually, documentation said it's an optional field, so this is not necessary, sorry!
"This field is informational, and has no impact on chart version calculations."

Reply:
Np, still worth adding!